src/ngraph/frontend/onnx_import/utils/common.hpp

```diff
@@ -67,28 +68,30 @@ namespace ngraph
                 return range;
             }
 
-            /// \brief      Handle negative axis value.
+            /// \brief      Handle out of range axis.
             ///
+            /// \param[in]  node        The node with requested axis.
             /// \param[in]  axis        The requested axis value.
-            /// \param[in]  tensor_dim  The corresponding tensor dimensionality.
+            /// \param[in]  tensor_rank  The corresponding tensor rank.
             ///
-            /// \tparam     T           Provided axis value type.
+            /// \return     If negative axis, then return sum of tensor rank and axis.
```

Comment:
What is the `sum of tensor rank and axis`? Please add a note that it's as if you were counting the axes from the end.